app/controllers/concerns/course/discussion/posts_concern.rb

```diff
@@ -56,7 +75,7 @@ def auto_subscribe_topic
   # When your model uses 'acts_as :topic', you can write: 'your_instance.topic' in this method.
   #
   # @return [Course::Discussion::Topic] The discussion topic record.
-  def topic
+  def discussion_topic
```

Comment:
Do you mean that I should modify the name `topic` in my concrete controller? Then I think it still does not follow the convention of namespacing. Can I use `forum_topic` in my concrete controller?